### ISSUE TITLE
mkdir: can't create directory because of permission denied

### ISSUE THREAD

**Issue by jobcespedes** [openebs/helm-operator#7]
### Issue 
No storage is created when using openebs-hostpath StorageClass. 

### Actual result 
The container init-pvc-&lt;uuid&gt; fails with:

```bash
mkdir: can't create directory '/data/pvc-3f7500d7-16ae-11ea-971b-fa163e796691': Permission denied
```

### Expected result
Storage created

### Workaround: 
1. Permissive selinux in nodes
2. Modify init-pvc-&lt;uuid&gt; and edit: `securityContext: {}`:

```yaml
spec:
  securityContext: 
    privileged: true
```

### Environment
Openshift: 4.2
openebsoperator: v1.3.0

### Other
I think the issue here is that init-pvc-&lt;uuid&gt; is not running as privileged. Running it as privileged is a better option that a permissive selinux. I just don't know yet which component is to be modified for that to happen.

**Comment by ksatchit** [comment]
cc: @kmova @ChandanSagar 

@jobcespedes , thank you for reporting this issue & apologies for the delay in response. While we shall look into this, would be great if you can add the workarounds into the description of the CSV here: https://github.com/openebs/helm-operator/tree/master/olm/1.3.0 & 1.4.0

**Comment by ksatchit** [comment]
This has been fixed in the 1.5 release with this PR: https://github.com/openebs/maya/pull/1542

With this, this step would be sufficient to successfully create the PV with openebs-hostpath storage class: 

![image](https://user-images.githubusercontent.com/21166217/71584240-d4b44100-2b37-11ea-8c70-55e426a05073.png)

Keeping this issue open until the OpenEBS 1.5 CSV is pushed to the OperatorHub. cc: @ChandanSagar

**Comment by ksatchit** [comment]
The OpenEBS 1.5 community operator is now out! @jobcespedes can we close this issue?
